.github/workflow/linters.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           node-version: "12.x"
       - name: Setup Webhint
         run: |
-          npm install --save-dev hint@6.x
+          npm install --save-dev hint@7.x
```

Comment:
>- [ ] Kindly, correct the linters path to your project. In the root project, you need to have a folder named .github. Inside this one, you need another folder named `workflows` not `workflow`. The linters needed for this project can be found [here](https://github.com/microverseinc/linters-config/tree/master/html-css);
>Here is a video demo I created demostrating  the above:
>
https://user-images.githubusercontent.com/52934128/164221056-46f41b39-cb1e-4fd9-a7f9-dd47bf835439.mp4
- [ ] Kindly ensure the Linter result is green. AKA: Linter should report NO errors.
and there should be no custom changes to linter config (as an attempt to cheat linter error reports).